Makefile

```diff
@@ -79,6 +79,9 @@ cross: build ## cross build the binaries for darwin, freebsd and\nwindows
 win: build ## cross build the binary for windows
 	$(DOCKER_RUN_DOCKER) hack/make.sh win
 
+darwin: build ## cross build the binary for OS X
+	$(DOCKER_RUN_DOCKER) hack/make.sh darwin
```

Comment:
Wonder, would simply doing this work? (untested)

    $(DOCKER_RUN_DOCKER) DOCKER_CROSSPLATFORMS="darwin/amd64" hack/make.sh cross

That would make it a lot easier to maintain (my worry is that `hack/make/darwin` gets out of sync with `hack/make/cross`